DESCRIPTION

```diff
@@ -38,11 +38,15 @@ Depends: R (>= 4.2)
 Imports: 
     admiraldev,
     dplyr (>= 1.0.0),
+    memoise,
```

Comment:
At Roche we had a bad experience with memoise, though in our case we had to cached dozens of large datasets. Let's keep this for now and see hot it goes.

Reply:
Yeah, let us use it judiciously. :)